changelogs/Spyder-6.md

```diff
@@ -172,6 +172,32 @@ In this release 25 pull requests were closed.
 
 ----
 
+## Version 6.0.7 (2025/05/22)
+
+### Important fixes
+
+* Fix hard crash when checking conda for cached kernels.
+* Fix inline plots failing due to faulty `traitlets` versions.
```

Comment:
* Fix crash at startup on Windows when Conda is not available.
* Fix failure to show plots in the Plots pane due to faulty `traitlets` versions.